main/src/cgeo/geocaching/log/AbstractLoggingActivity.java

```diff
@@ -34,7 +34,11 @@ public boolean onCreateOptionsMenu(final Menu menu) {
 
         final SubMenu menuLog = menu.findItem(R.id.menu_templates).getSubMenu();
         for (final LogTemplate template : LogTemplateProvider.getTemplatesWithSignature()) {
```

Comment:
Just an idea: what about sorting template order in menu by template name?

Reply:
That'd remove the possibility for the user to decide about the order, I think the current is better than alphabetical order.